startup/BMM/functions.py

```diff
@@ -172,6 +172,7 @@ def clear_dashboard():
     rkvs.set('BMM:scan:type',      '')
     rkvs.set('BMM:scan:starttime', '')
     rkvs.set('BMM:scan:estimated', '')
+    return()
```

Comment:
Not sure it's needed. Do you intend to return an empty tuple?

Reply:
agreed.  Will remove